content/doc/book/using/remote-access-api.adoc

```diff
@@ -0,0 +1,260 @@
+---
+layout: section
+title: Remote Access API
+---
+
+
+Jenkins provides machine-consumable remote access API to its
+functionalities. Currently it comes in three flavors:
+
+. XML
+. JSON with JSONP support
+. Python
+
+Remote access API is offered in a REST-like style. That is, there is no
+single entry point for all features, and instead they are available
+under the `+".../api/"+` URL where `+"..."+` portion is the data that it
+acts on.
+
+For example, if your Jenkins installation sits at http://ci.jruby.org/,
```

Comment:
I would recommend using https://ci.jenkins.io in the examples, this is long dead

Reply:
ok